src/Widgets/NavigationWidget/SubComponents/CurrentUserDropdown.tsx

```diff
@@ -68,13 +70,16 @@ export const CurrentUserDropdown = connect(function CurrentUserDropdown({
       ) : null}
 
       <NavDropdown.Item
-        as={LinkTag}
         eventKey="MetaNavigation-LogOut"
         key="MetaNavigation-LogOut"
-        onClick={() => {
-          logout()
+        onClick={async () => {
+          const target = await load(() => {
+            const root = Obj.root()
+            return root ? urlFor(root) : undefined
```

Comment:
* [ ] FYI If we move the `urlFor` loading out of the click handler the button will be more responsive on clicking and the `onClick` callback would be simpler.